pandas/tests/io/formats/test_to_csv.py

```diff
@@ -223,3 +224,46 @@ def test_to_csv_multi_index(self):
 
         exp = "foo\nbar\n1\n"
         assert df.to_csv(index=False) == exp
+
+    def test_to_csv_compression(self):
+        import gzip
+        import bz2
```

Comment:
I think we can just import at the top.  You're not polluting the namespace with a `from` import.  Also, reference the issue that you mention in the discussion under the function definition.